### PR TITLE
Henry Week 5 Front-End Work

### PR DESCRIPTION
**Pull request by Henry-Graves**
- Add prototype table and chart.
- Add prototype search bar, shows a specific person's contribution page (chart page) for now, but should probably dynamically modify the table displayed later on.
- Improve clarity of message in header paragraph. 
- Fix search bar button size. 
- Remove irrelevant code that displays a random greeting (the code that existed by default in this project).

image

image

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -2,18 +2,92 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Political Contributions Tracker</h1>
+
+      <p>Welcome to the prototype of the Political Contributions Tracker.
+        <br>
```
Comment (Priya411) [inline src/main/webapp/index.html:16]: Can you add a TODO to update this part for the coming week?

**Review by Priya411 — COMMENTED** [review]
Good work! Just added a small comment but looks good otherwise.

File: src/main/webapp/index.html (hunk 1 above)
Comment (Henry-Graves) [reply src/main/webapp/index.html:16]: @Priya411 Done, please take a look at lines 13, 25, and 36 in index.html for the added TODO comments.

**Comment by vanessa-vasquez** [comment]
This looks good! Everything seems clear to me. I'm excited to start working on it!

**Review by vanessa-vasquez — APPROVED** [review]

**Review by Priya411 — APPROVED** [review]

**Pull request merged**